docs/developer/running-saleor/configuration.mdx

```diff
@@ -185,6 +185,19 @@ Controls [Django’s secret key](https://docs.djangoproject.com/en/3.0/ref/setti
 
 Controls the production assets mount path. Defaults to `/static/`.
 
+### `JWT_EXPIRATION_DELTA`
+
+Controls the time which defines how long the JWT authentication token will be valid. The value should be provided in seconds. Defaults to 5 minutes.
+
+
+### `JWT_REFRESH_EXPIRATION_DELTA`
+
+Controls the time which defines how long the JWT refresh token will be valid. The value should be provided in seconds. Default to 30 days.
```

Comment:
Not seconds, but `timedelta` objects

Reply:
`seconds`

Reply:
```
JWT_EXPIRATION_DELTA = timedelta(
    seconds=os.environ.get("JWT_EXPIRATION_DELTA", 5 * 60)  # type: ignore
)
```

Reply:
Ah nevermind, I see we changed it in the PR https://github.com/mirumee/saleor/blob/877810e83da772569f51ac5650edd11ceed4f629/saleor/settings.py#L552-L561

Reply:
Yeap, we drop the graphql_jwt package.